.gitignore

```diff
@@ -64,6 +64,9 @@ test-results/
 !.yarn/versions
 development/generate-attributions/.yarn/*
 
+# MetaMask
+.metamask/*
```

Comment:
I cache the foundry foundries in a new `.metamask` root folder. we want to ignore it.

Maybe this is where I could put the foundry version? instead of the package.json? hmmmm. not sure I like that either